### ISSUE TITLE
False positive for W0105

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#193]
Originally reported by: **the mulhern (BitBucket: [the_mulhern](http://bitbucket.org/the_mulhern))**

---

Mini example:

```
#!python

class Junk(object):

    def __init__(self, priority, *args, **kwargs):
        self.priority = priority
        """a legal attribute docstring"""
```

Pylint reports:

W0105(pointless-string-statement) String statement has no effect

for """a legal attribute docstring"""

But, according to PEP 257 http://legacy.python.org/dev/peps/pep-0257/#what-is-a-docstring) this line is an "attribute docstring" and therefore a legal docstring, not a pointless string statement.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/193
